definitions/firefox_desktop.toml

```diff
@@ -1498,6 +1498,74 @@ Count of events where the client changed the weather widget display type to Simp
 
 ########################### END OF NEW TAB METRICS ###########################
 
+########################### START OF AD FUNNEL METRICS ###########################
+[metrics.newtab_clients_with_sponsored_tile_impressions]
+data_source = "newtab_clients_daily"
+select_expression = "COALESCE(COUNTIF(sponsored_topsite_tile_impressions>0), 0)"
+friendly_name = "Newtab Clients with Sponsored Tile Impressions"
+type = "scalar"
+description = """
+Count of clients with any sponsored tile impressions.
+"""
+
+[metrics.newtab_clients_with_sponsored_tile_clicks]
+data_source = "newtab_clients_daily"
+select_expression = "COALESCE(COUNTIF(sponsored_topsite_tile_clicks>0), 0)"
+friendly_name = "Newtab Clients with Sponsored Tile Clicks"
+type = "scalar"
+description = """
+Count of clients with any sponsored tile clicks.
+"""
+
+[metrics.frac_users_with_sponsored_tile_impressions]
+depends_on=['newtab_sponsored_tiles_enabled', 'newtab_clients_with_sponsored_tile_impressions']
+friendly_name = "Fraction of Users with Sponsored Tile Impressions"
+description = """
+Fraction of users with sponsored tile impressions as a fraction of users with sponsored tiles enabled
```

Comment:
just to make sure the fraction is correctly computed, right now as it's defined it will `newtab_sponsored_tiles_enabled / newtab_clients_with_sponsored_tile_impressions`, which doesn't match quite with this description